python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -207,6 +208,18 @@ def test_range(self):
             .equals(self.spark.range(start=0, end=10, step=3, numPartitions=2).toPandas())
         )
 
+    def test_create_global_temp_view(self):
+        # SPARK-41127: test global temp view creation.
+        with self.tempView("view_1"):
+            self.connect.sql("SELECT 1 AS X LIMIT 0").createGlobalTempView("view_1")
+            self.connect.sql("SELECT 2 AS X LIMIT 1").createOrReplaceGlobalTempView("view_1")
+            self.assertTrue(self.spark.catalog.tableExists("global_temp.view_1"))
+
+            # Test when creating a view which is alreayd exists but
+            self.assertTrue(self.spark.catalog.tableExists("global_temp.view_1"))
+            with self.assertRaises(_MultiThreadedRendezvous):
```

Comment:
cc @zhengruifeng @HyukjinKwon 

I found this is what we have now when there is an exception happened during a RPC call.

I guess we need to improve it.

Reply:
`_MultiThreadedRendezvous` is raised from gRPC channel when gRPC finds response state code is not `OK` (thus some error has happened).

Reply:
Yes, right now, the client does not do any error handling, the exception simply bubbles up.

Reply:
you can just check for `grpc.RpcError`

Reply:
@grundprinzip yes! It is much better to use `grpc.RpcError` for readability.

Reply:
Eventually we should also use error classes in client side (based on the grpc error to map to error classes by a way). Right now we only have this.